ruby-challenges.rb

```diff
@@ -22,7 +38,14 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def no_vowels (string)
+  string_array = string.split('')
+  string_array.delete('a','e','i','o','u')
+  string_array.join('')
+end
+p no_vowels
```

Comment:
Good ideas here, Zack.  You're just going about it the JavaScript way.  I encourage you to redo this one using ruby methods.